dpdata/pymatgen/molecule.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+from pymatgen.core import Molecule
+from collections import Counter
+
+def to_system_data(file_name) :
+    mol = Molecule.from_file(file_name)
+    elem_mol = list(str(site.species.elements[0]) for site in mol.sites)
+    elem_counter = Counter(elem_mol)
+    atom_names = elem_counter.keys()
+    atom_numbs = elem_counter.values()
+
+    atom_types = []
+    for idx,ii in enumerate(atom_numbs) :
+        for jj in range(ii) :
+            atom_types.append(idx)
```

Comment:
It seems that you are confirming the atom type is something like `0 0 0 0 1 1 1 2 2 2` . Will it be possible for something like `0 0 1 1 0 0 1 2 2 2` ?

Reply:
Changed!